### PR TITLE
mu-repo 1.8.1 (new formula)

### PR DESCRIPTION
**Pull request by bobsoppe**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mu.rb [hunk 1]
```diff
@@ -48,6 +48,8 @@ class Mu < Formula
 
   fails_with gcc: "5"
 
+  conflicts_with "mu-repo", because: "both install `mu` binaries"
+
```
Comment (bobsoppe) [inline Formula/mu.rb:52]: Do I need to add this `conflicts_with` block in this commit/pull request or a separate one?

Comment (carlocab) [reply Formula/mu.rb:50]: Let's add the conflicts after this PR is merged.

**Review by carlocab — APPROVED** [review]
Thanks, @bobsoppe!

**Comment by bobsoppe** [comment]
No problem! CI also seemed to be finished

**Comment by carlocab** [comment]
Should be merged now. When you open a new PR to add the `conflicts_with` lines, please make sure to have two separate commits: one for `mu` and another for `mu-repo`, with the respective formula name first in each commit message. Thanks!

**Pull request closed without merging**